CHANGES.md

```diff
@@ -7,6 +7,8 @@ versioning](https://semver.org/) and all releases are available on
 
 ## 0.6.0 —
 
+- {gh}`487` Replace pydot_layout with pygraphviz_layout. ({ghuser}`lars-reimann`).
```

Comment:
I was just about to comment that the convention is to add at the bottom, but this way is actually much better because it follows the git log.

@ChristianZimpelmann, can you have an RA change the order and add expand the todo "Document PR in CHANGES.md." into "Document PR in CHANGES.md just below the appropriate target version".